Update runtime to 6.10

PunkPangolin opened the pull request.

PunkPangolin committed:
Update runtime to 6.10

jorio commented:
Hey @PunkPangolin, thank you for this!

I'd like to do some testing before publishing this update - in the past, GNOME users have [lost their dark themes in GitFourchette](https://github.com/jorio/gitfourchette/issues/51), because [org.kde.KStyle.Adwaita](https://github.com/flathub/org.kde.KStyle.Adwaita) typically lags behind the latest available Qt runtime. I'd just like to make sure we don't merge this too soon to prevent this issue from surfacing again. I'll try to have a look this weekend!

PunkPangolin commented:
> Hey @PunkPangolin, thank you for this!
> 
> I'd like to do some testing before publishing this update - in the past, GNOME users have [lost their dark themes in GitFourchette](https://github.com/jorio/gitfourchette/issues/51), because [org.kde.KStyle.Adwaita](https://github.com/flathub/org.kde.KStyle.Adwaita) typically lags behind the latest available Qt runtime. I'd just like to make sure we don't merge this too soon to prevent this issue from surfacing again. I'll try to have a look this weekend! 

You are welcome! I am using GNOME, just tested that and dark mode does indeed not work. `org.kde.KStyle.Adwaita` is also available for 6.10 yet, so I guess this is something for later.

PunkPangolin commented:
bot, build

PunkPangolin commented:
bot, build

jorio commented:
I opened a PR to update org.kde.KStyle.Adwaita's runtime (https://github.com/flathub/org.kde.KStyle.Adwaita/pull/23). Hopefully they'll merge it and then we can update the runtime for GitFourchette.

ghost commented:
Seems merged.

jorio committed:
/app/bin/git* symlinks must now point to /app/libexec/*

Fix GitFourchette doesn't detect the git install embedded in the Flatpak build with org.kde.Sdk 6.10.

jorio committed:
pygit2 1.19.1
